### ISSUE TITLE
Timestream as first class Database option

### ISSUE THREAD

**Issue by arkllc** [aws/aws-appsync-community#136]
Is your feature request related to a problem? Please describe.

All of my application is built completely on AppSync transform with DynamoDB as backend DB. A part of my application has to monitor the progress of a job and store the progress back into a data store via AppSync data mutation. The subscribers to the mutation will automatically get notified. Before TimeStream I was thinking of storing the time series data of progress (timestamp and %of job complete) on a given job in DynamoDB. However, Timestream has some great features of tiering and it will also reduce the burden of me doing this with extra coding with DynamoDB Streams. AppSync is a perfect fit here as it can provide low coupling due to a facade behind which I can abstract various purpose-built database types - Dynamodb and TimeStream. Please let me know if you need further clarification.

Describe the solution you'd like
TimeStream should be supported as a first-class DB just like DynamoDB and Aurora

Describe alternatives you've considered
The alternative is to use DynamoDB. However, for a part of my DB where data shape is more like TimeSeries, it would be nice to use TimeStream

**Comment by jonsmirl** [comment]
I agree with this. Meanwhile you can write a lambda resolver which will let you integrate Timestream. In my use case It is IOT data going to Timestream and I want to display pretty charts. My need would be read-only timestream query access. There is too much data flowing to run it all through appsync for the writes. I'd like to, but I'd probably get a $1M AWS bill if I did it.

**Comment by vitalbone** [comment]
I'd love to see TimeStream supported as an official storage type. Our use case comes from an event tracking requirement. We need to log time-series data whenever users interact with various parts of the application.

It'd be great to be able to spin up a TimeStream instance along with the rest of our Amplify (and CloudFormation) infrastructure.
